### DISCUSSION TITLE
Storybook is not working with Vite 2.9.12

### DISCUSSION THREAD

**Discussion by satyendera** [storybookjs/builder-vite#480] [Q&A] [answered]
i am using @storybook/builder-vite within Vite app , but when i start storybook it shows me error - ERR! TypeError: vite.createFilter is not a function
ERR!     at viteReact (/Users/satyenderasharma/Desktop/projects/VTS/my-app/node_modules/@storybook/builder-vite/node_modules/@vitejs/plugin-react/dist/index.cjs:202:21
![Uploading Screen Shot 2022-08-16 at 11.09.16 AM.png…]()

**Comment by IanVS** [comment, marked as answer]
Hi, the latest versions of this project are only compatible with vite 3.  If you'd like to use it with vite 2, please pin to version `0.1.x` in your package.json.

**Comment by satyendera** [comment]
thanks , yes latest version solved this problem
